### DISCUSSION TITLE
Measuring speed of execution

### DISCUSSION THREAD

**Discussion by TrueCherchor** [geeksblabla/blanat#15] [Q&A] [answered]
Salam community,
Thank you for this initiative. 
I am willing to participate in this contest and I have basically two questions:
1.  How will you measure the speed of execution of a given code?
2. Will the choice of the programming language affect the speed of execution like in the case of interpreted language (java) vs compiled ones (C or C++)

Thank you.

**Comment by aboullaite** [comment, marked as answer]
I believe answer to both questions can be found here https://github.com/chermehdi/blarun/blob/6dbe25cfaff26a143d6260172c04bf79fa59e4ea/src/main.rs#L119 😅 nothing better than reading the source code right 👀

**Reply by TrueCherchor** [reply]
Thank you for your reply.  Which source code do you mean?

**Reply by aboullaite** [reply]
The one I mentionned in the reply :)
